.github/workflows/npm-test.yml

```
@@ -0,0 +1,26 @@
+name: Node.js CI
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  build:
```

Let's rename the job:

```suggestion
  test:
```